app/services/calculate_lessons_rating.rb

```diff
@@ -0,0 +1,38 @@
+class CalculateLessonsRating
```

Comment:
It's better to name CalculateLessonsRatingService to make understandable, that this is a service.